packages/inputs-minimal/src/radio_group/index.jsx

```diff
@@ -24,33 +25,42 @@ const RadioGroup = ({
   items,
   name,
   value,
+  error,
   Layout = RadioGroupHorizontalLayout,
 }) => {
   const handleChange = (event, val) => {
     onChange(event, val);
   };
 
   return (
-    <Layout>
-      {items.map((item) => (
-        <Radio
-          key={item.value}
-          onChange={handleChange}
-          name={name}
-          label={item.label}
-          value={item.value}
-          disabled={item.disabled}
-          checked={value === item.value}
-        />
-      ))}
-    </Layout>
+    <div>
```

Comment:
I think if comment bellow will be implemented - there won't be a need for div. 
If not - isn't it worth having a Fragment instead? `<>...</>`

Reply:
https://github.com/goodhood-eu/goodhood/pull/161#discussion_r911696546